src/graph/planner/plan/Query.h

```diff
@@ -1555,7 +1555,7 @@ class AppendVertices final : public GetVertices {
                     nullptr,
                     false,
                     {},
-                    0,
+                    -1,  // means no limit
```

Comment:
`IndexScan` use `9223372036854775807` indicate no limitation.

Reply:
Negative number is more easy to check is there limit requirement?